torch/overrides.py

```diff
@@ -422,7 +422,7 @@ def get_testing_overrides() -> Dict[Callable, Callable]:
         torch.isnan: lambda input: -1,
         torch.istft: (lambda input, n_fft, hop_length=None, win_length=None, window=None, center=True,
                       normalized=False, onesided=None, length=None, return_complex=False: -1),
-        torch.kl_div: lambda input, target, size_average=None, reduce=None, reduction='mean', log_target=False: -1,
+        torch.kl_div: lambda input, target, size_average=None, reduce=None, reduction='mean', log_target=False: -1,  # type: ignore[attr-defined]
```

Comment:
`kl_div` and other functions in the main namespace do exist, so instead of adding ignores here, these should be added to codegen so the annotations land in `torch/_C/_VariableFunctions.pyi`